pyscriptjs/tsconfig.json

```diff
@@ -1,5 +1,7 @@
 {
-  "extends": "@tsconfig/svelte/tsconfig.json",
+  "$schema": "https://json.schemastore.org/tsconfig",
```

Comment:
I had to replace this with values which were taken from the source `npm` package: https://www.npmjs.com/package/@tsconfig/svelte

This is because, while the import could be found locally when packages were installed in `node_modules`, it could not be found while `pre-commit` was running.